server/src/room.ts

```diff
@@ -7,13 +7,15 @@ export default class Room {
     gameStarted: boolean;
     numWords: number;
     playersFinished: number;
+    maxCapacity: number;
     
     constructor(roomID: string) {
         this.roomID = roomID;
         this.#players = new Map()
         this.gameStarted = false;
         this.numWords = 0;
         this.playersFinished = 0;
+        this.maxCapacity = 1;
```

Comment:
was this just 1 for testing?

Reply:
Yes, once this is approved, I'll change it back to 4